src/page/Page.tsx

```diff
@@ -46,7 +46,7 @@ export const NeoPage = ({
   const loadingMessage = <div>Loading card...</div>;
   const [isDragging, setIsDragging] = React.useState(false);
   const [layouts, setLayouts] = React.useState(defaultLayouts);
-  const [lastElement, setLastElement] = React.useState(<div key={getReportIndex(pagenumber, 999999)}></div>);
+  const [lastElement, setLastElement] = React.useState(<div key={getReportIndex(pagenumber, '999999')}></div>);
```

Comment:
i don't think the concept of lastElement makes sense anymore

Reply:
Same as usual